virttest/utils_netperf.py

```diff
@@ -75,14 +76,7 @@ def __init__(self, address, netperf_path, md5sum="", netperf_source="",
         self.netperf_base_dir = self.remote_path
         self.netperf_file = os.path.basename(self.netperf_source)
 
-        if client == "nc":
-            self.prompt = r"^\w:\\.*>\s*$"
-            self.linesep = "\r\n"
-            self.status_test_command = "echo %errorlevel%"
-        else:
-            self.prompt = "^\[.*\][\#\$]\s*$"
-            self.linesep = "\n"
-            self.status_test_command = "echo $?"
+        if client == "ssh":
```

Comment:
ssh has different decomp_cmd for different pack_suffix(".tar.gz" or ".tar.bz2").